packages/core/src/Stepper.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Flex from './Flex'
+import Step from './Step'
+
+const Container = styled(Flex)`
```

Comment:
I would avoid using the name "Container" as the there's indeed a DS component called `Container`. Maybe use a name that retains the primitive `Flex` such as `FlexWrapper` or something similar?